update_repos

```diff
@@ -97,7 +97,7 @@ def read_api_uri(uri, config, page=1):
         print("Trying URI {} with headers {}".format(uri, headers))
 
     req = Request(uri, headers=headers)
-    return urlopen(req).read().decode()
+    return urlopen(req).read().decode('utf-8')
```

Comment:
Yeah, I agree with that. We're already assuming that the GitHub API is sending back text and I'm pretty sure we can assume it's going to be utf-8, although I don't know if that's specified anywhere.